server/integration-tests/sqlite_test.hs

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env stack
```

Comment:
What is this file supposed to test? I think we can assume that SQLite works. Is this a leftover experiment?

Reply:
This was indeed an experiment. It was actually pretty useful to try a few different approaches on how to store the value in Sqlite. I could move it somewhere else, but I think it doesn't really hurt to leave it here.